packages/jest-diff/src/index.ts

```diff
@@ -8,17 +8,13 @@
 import prettyFormat = require('pretty-format');
 import chalk from 'chalk';
 import getType = require('jest-get-type');
-import {
-  DIFF_DELETE,
-  DIFF_EQUAL,
-  DIFF_INSERT,
-  Diff as DiffClass,
-} from './cleanupSemantic';
+export {DIFF_DELETE, DIFF_EQUAL, DIFF_INSERT, Diff} from './cleanupSemantic';
```

Comment:
I'd prefer pulling the exports out of the import list, it's very hard to see what (or that something) is exported here